doc/keepalived.conf.SYNOPSIS

```diff
@@ -95,6 +95,7 @@ global_defs {                # Block identification
                                            # (in seconds, resolution microseconds)
     vrrp_mcast_group4 <IPv4 ADDRESS>       # optional, default 224.0.0.18
     vrrp_mcast_group6 <IPv6 ADDRESS>       # optional, default ff02::12
+    vrrp_skip_check_adv_addr <BOOL>        # ?
```

Comment:
This is documented in docman/man5/keepalived.conf.5 so please copy that description.

It would also be good to add `skip_check_adv_addr` in the vrrp_instance block.